index.html

```diff
@@ -6,22 +6,51 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Awesome Books</title>
     <link rel="stylesheet" href="styles.css" />
+    <link
+      rel="stylesheet"
+      href="	https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/css/bootstrap.min.css"
+    />
   </head>
   <body>
     <header></header>
     <main>
       <section id="awesome-books">
-          <h1>Awesome Books List</h1>
-          <div id="books-list"></div>
+        <h1 class="text-center fw-bold pt-3 pt-lg-5 pb-3">All awesome books</h1>
+        <div class="container-fluid">
+          <div
+            class="awesome-container row px-2 px-md-3 px-lg-5"
+            id="books-list"
+          ></div>
+        </div>
       </section>
 
       <section id="input-form">
-          <form action="#" id="add-book-form">
-              <h2>Add a new book</h2>
-              <input type="text" placeholder="Title" name="title" required>
-              <input type="text" placeholder="Author" name="author" required>
-              <input type="submit" value="Add">
-          </form>
+        <div class="container-fluid d-flex justify-content-center">
+          <div class="row">
+            <form action="#" id="add-book-form" class="col-12">
+              <h2 class="text-center pt-5 fw-bold">Add a new book</h2>
+              <input
+                type="text"
+                class="input-form mt-3 col-12 border-0 px-2 py-2"
+                placeholder="Title"
+                name="title"
+                required
+              />
+              <input
+                type="text"
+                class="input-form border-0 mt-2 col-12 px-2 py-2"
+                placeholder="Author"
+                name="author"
+                required
+              />
+              <input
+                type="submit"
+                class="border-0 mt-2 text-white btn bg-dark btn-sm col-7 py-2 mt-3"
+                value="Add"
+              />
```

Comment:
- [ ] It would be better if the general layout is followed as much as possible. For example, in your app submit button is placed on the left side while in the original given wireframe it is placed on the right side of the form. Kindly fix it. 👇 
- Your design 👇 
![Screenshot (286)_LI](https://user-images.githubusercontent.com/67087664/176176076-36a7e59e-1f67-4503-85e3-6c957da0b9c4.jpg)
- Original 👇 
![Screenshot (287)](https://user-images.githubusercontent.com/67087664/176176230-ca28df35-0fc5-42e1-a8bf-3a6eb872653a.png)